apps/hannk/util/buffer_util.h

```diff
@@ -6,6 +6,7 @@
 
 #include "HalideBuffer.h"
 #include "HalideRuntime.h"
+#include "interpreter/interval.h"  // for max_rank
```

Comment:
Not sure about including a comment like this. Fine to leave it but it's weird (we don't have that comment on any other include...).

Reply:
Removed